syntax = "proto3";

package website_builder;

message TestimonialItem {
  string text_i18n_key = 1;
  string author_i18n_key = 2;
  string img_src = 3;
  string img_alt_i18n_key = 4; // For accessibility and potential localization
}
